autoload/fern/internal/cursor.vim

```diff
@@ -18,6 +18,7 @@ if has('nvim')
 
     function! s:restore() abort
       set guicursor+=a:Cursor/lCursor
+      sleep 1m
```

Comment:
I tried putting `0m` here, but that didn't work. It seems like the amount of time doesn't matter, as long as Neovim has slept.